src/test/perf_counters.cc

```diff
@@ -48,6 +49,9 @@
 
 using namespace std;
 
+constexpr std::string_view empty_json_output_raw = R"({}
```

Comment:
Constants should be capitalized.  I also tweaked a name a bit to avoid confusion between `"format": "raw"` and `"format": "json"`:
```suggestion
constexpr std::string_view EMPTY_DUMP_FORMAT_RAW = R"({}
```

Reply:
This may be in some guidelines doc, but I think that's just a throwback or tic to old C coding style.

Reply:
We generally follow the Google C++ style guide which says:
> Variables declared constexpr or const, and whose value is fixed for the duration of the program, are named with a leading "k" followed by mixed case.

but there is an exception in Ceph: https://github.com/ceph/ceph/blob/7800ea59aa7d03dd22942e13b481f2a49432e085/CodingStyle#L59

Either way, constants are meant to stand out and it's useful IMO.

Reply:
anything but `kSomeThing` 😆

Reply:
that's how you know its konst